localstack/services/events/scheduler.py

```diff
@@ -25,7 +61,10 @@ def run(self):
 
     def should_run_now(self):
         schedule = CronTab(self.schedule)
-        delay_secs = schedule.next()
+        delay_secs = schedule.next(
+            default_utc=True
```

Comment:
good catch 👍 
This might be also worth mentioning in the [docs](https://docs.localstack.cloud/user-guide/aws/events/).

The announcement blog post for "EventBridge Scheduler" has a nice table comparing "Amazon EventBridge Scheduler" and "Amazon EventBridge rules" (this PR).
The "EventBridge Scheduler" supports time zones, but "EventBridge rules" does not.
> You can use EventBridge Scheduler to start and stop all the thousands of instances and also respect time zones.
https://aws.amazon.com/blogs/compute/introducing-amazon-eventbridge-scheduler/

Can you link that post and write a short summary to distinguish these schedulers in the backlog item and roadmap item? I remember we ran into this confusion before, so I think it's worth clarifying this.